### ISSUE TITLE
Suggestions for issues: ( that can help beginners) 🔰 

### ISSUE THREAD

**Issue by soumik2012** [Tenet-Coding/Hacktoberfest-Projects#11]
## Suggestions for issues: ( that can help beginners)
- brief project description

- use cases

- additional challenges (that are optional)

- reference images (warn that is just a reference and encourage their own designs)

- requirements to approve pull request Added 2 mini projects #18 (comment)

### clock example: #1

> quick examples that can be improved

description: Make A clock, a device used to measure, keep, and indicate time
...
...

use cases: users can see the current time in a clock
.....
.....

additional challenges:

implement the function to change between digital and analog clock
implement the function to change between 12-hour am-pm clock | 24-hour military time (digital edition)
show current day
reference images:

clock

requirements: ....

### mini projects sugestions:

- [ ]  - cash register
- [ ] - phone number validate
- [ ] - classic todo app
- [ ]  - notepad that saves notes in localStorage
- [ ]  - image crop
- [ ]  - landing page
- [ ]  - recent news app (from API)

hope this help, thanks 👍

**Comment by siddhantkhariwal** [comment]
Hey, Can I fix this issue?

**Comment by soumik2012** [comment]
> Hey, Can I fix this issue?

okay, which mini-project do you want to work on?

**Comment by nandan-shah** [comment]
hey, can I work on todo or news app?

**Comment by soumik2012** [comment]
> hey, can I work on todo or news app?

Yes @nandan-shah you can work on it create an issue I will assign to you.

**Comment by soumik2012** [comment]
Yup @Sugyanee you can!!  please create an issue I will assign to you.

**Comment by siddhantkhariwal** [comment]
> > Hey, Can I fix this issue?
> 
> okay, which mini-project do you want to work on?

I can work on the notepad app.

**Comment by Harshil-Jani** [comment]
> > > Hey, Can I fix this issue?
> > 
> > 
> > okay, which mini-project do you want to work on?
> 
> I can work on the notepad .

@siddhantkhariwal Go Ahead ! Create Issue for the same. You will be assigned.
Happy HacktoberFest :tada:

**Comment by SANTHOSH17-DOT** [comment]
Can I work on phone number validator??

**Comment by Zackaria-Slimane** [comment]
Hey there ! 
happy to work on the following if possible 
 - cash register
- classic todo app
- notepad that saves notes in localStorage
- recent news app (from API)

Please let me know what's available to work on :)

**Comment by 26tanishabanik** [comment]
@soumik2012 I want to work on the image crop project, can you kindly explain that whether it will be a GUI?

**Comment by 26tanishabanik** [comment]
> @soumik2012 I want to work on the image crop project, can you kindly confirm that whether it will be a GUI?

@soumik2012 the script is ready for image cropper using OpenCV and Tkinter, can I make a pull request?

**Comment by 26tanishabanik** [comment]
I have made a pull request for the Image cropper project, issue no #61 , pull request no #62

**Comment by 26tanishabanik** [comment]
#62

**Comment by saugat-rimal** [comment]
hey, can I work on todo

**Comment by bonvagnik** [comment]
May I work on a new project- an expense tracker using ReactJS?

**Comment by ved08** [comment]
hey @soumik2012 can I work on phone number validator?

**Comment by NaimaGhulamMuhammad** [comment]
hey @soumik2012, I want to add a random number of colors in the background changer as there are only four colors changing...

**Comment by soumik2012** [comment]
These are the issue already made. You create own unique project and add them.

**Comment by prerna2712** [comment]
Can I work on the cash register?

**Comment by soumik2012** [comment]
@prerna2712 yes you can go ahead make an issue will assign to you!!

**Comment by SohanaShabnam05** [comment]
Hey, please can I work on classic to-do app??
please assin me for this.
